src/encoding.md

```diff
@@ -344,6 +345,57 @@ fn run() -> Result<()> {
 # quick_main!(run);
 ```
 
+[ex-invalid-csv]: #ex-invalid-csv
+<a name="ex-invalid-csv"></a>
+## Handle invalid CSV data with Serde
+
+[![csv-badge]][csv] [![serde-badge]][serde] [![cat-encoding-badge]][cat-encoding]
+
+CSV files often contain invalid data. For these cases, the csv crate
+provides a custom deserializer, [`csv::invalid_option`], which automatically
+converts invalid data to None values.
+
+```rust
+# #[macro_use]
+# extern crate error_chain;
+extern crate csv;
+extern crate serde;
+#[macro_use]
+extern crate serde_derive;
+
+#[derive(Debug, Deserialize)]
+struct Record {
+    name: Option<String>,
```

Comment:
do the Option's here make sense without `csv::invalid_option`? these might obfuscate the example.